src/EditorFeatures/CSharpTest/CodeActions/IntroduceVariable/IntroduceVariableTests.cs

```diff
@@ -5718,6 +5718,31 @@ void Goo()
     {
         Bar(1, Bar(V));
     }
+}");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsIntroduceVariable)]
+        [WorkItem(27949, "https://github.com/dotnet/roslyn/issues/27949")]
+        public async Task TestWhitespaceSpanInAssignment()
+        {
+            await TestMissingAsync(@"
+class C
+{
+    int x = [| |] 0;
```

Comment:
I haven't read the 9000 lines of pre-existing tests, but are there any with spans like this that also end touching the next token? e.g. `x = [| |]0;` I mentioned this case elsewhere, and I'm not sure if the behavior (no refactoring offered) is intentional or not.

Reply:
Also, are there test cases similar to this one where the span is in whitespace (just like this) but the leading token contains, additionally, a `/**/` so that the check for all trivia being whitespace on line 198 of the service goes the opposite way?